File: backend/src/main/java/julien/maillard/model/StatisticDTO.java
package julien.maillard.model;

import java.util.List;
import lombok.Getter;
import lombok.Setter;
import lombok.AllArgsConstructor;
import lombok.NoArgsConstructor;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class StatisticDTO {

    private List<List<Integer>> developers;
    private List<List<Integer>> recruiters;
    private List<List<Integer>> students;
    private List<List<Integer>> clients;
    private List<List<Integer>> curious;
    private List<List<Integer>> others;

}

File: frontend/src/app/models/StatisticDTO.ts
export class StatisticDTO {

    developers: number[][];
    recruiters: number[][];
    students: number[][];
    clients: number[][];
    curious: number[][];
    others: number[][];

    constructor(developers: number[][], recruiters: number[][], students: number[][], clients: number[][], curious: number[][], others: number[][]) {
        this.developers = developers;
        this.recruiters = recruiters;
        this.students = students;
        this.clients = clients;
        this.curious = curious;
        this.others = others;
    }
}